pkg/client/core/store/filesystem/container_repository_filesystem.go

```diff
@@ -0,0 +1,62 @@
+package filesystem
```

Comment:
You can keep it if you want to, but I will remove it in my state PR, so if you waaaant to 😬